docs/docs/sr.md

```diff
@@ -0,0 +1,72 @@
+# Quantum Geometric Tensor ( and Stochastic Reconfiguration )
+
+The Quantum Geometric Tensor (QGT) is the Fubini-Study metric tensor of the manifold on which a variational state is defined.
+In general it is not possible to compute a form of the QGT valid on the whole manifold, but we can compute local approximations valid in small region around a given point $ W $
+To give an example, we consider a variational wavefunction $ \psi : \mathbb{R} \rightarrow \mathscr{H} $ which maps elements of the variational manifold $ W $ to vectors in the Hilbert space (wavefunctions) $ \psi_W $.
+
+The variational manifold has a simple Euclidean metric, therefore the distance between two points $ \bf{W} $ and $\bf{W} + \bf{\delta W} $ is simply  $ \bf{\delta W} $.
+However, what we really are interested in when optimizing variational ansatze is not the Euclidean distance between those two poins, but rather the distance in the Hilbert space between the elements in the co-domain.
+As the objects $\psi_{\bf{W}}$ is defined on an Hilbert space, the correct metric there is $ L_2 $ and not the Euclidean norm, we consider the distance defined as
```

Comment:
You mean parameter space when you say "variational manifold" sometimes, right? The _variational manifold_ is almost everywhere defined as $M = \{\psi_W | W \in C^M \}$ and is thus part of the Hilbert space and definitely not Euclidean.
```suggestion
To give an example, we consider a variational ansatz $ \psi\colon \mathbb{R} \rightarrow \mathscr{H} $ which maps elements $W$ of the parameter space to vectors in Hilbert space (quantum states) $\psi_W$.

The parameter space has a simple Euclidean metric, therefore the distance between two points $ \bf{W} $ and $\bf{W} + \bf{\delta W} $ is simply  $ \bf{\delta W} $.
However, what we really are interested in when optimizing variational ansätze is not the (Euclidean) distance between those two points in parameter space, but rather the distance in the Hilbert space between the corresponding quantum states (which also properly takes into account gauge degrees of freedom).
As the objects $\psi_{\bf{W}}$ is defined on an Hilbert space, the correct metric there is $ L_2 $ and not the Euclidean norm, we consider the distance defined as
```
The last paragraph seems incomplete, so I've left it as is for now.

Reply:
Yeah i'm still writing this...
Yes. I meant parameter space, thanks for catching this.